src/index.js

```diff
@@ -1,2 +1,54 @@
 /* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension */
+/* eslint linebreak-style: ["error", "windows"] */
 /* @jsx createElement */
+
+function createElement(tagName, props, ...children) {
+  const element = document.createElement(tagName);
+
+  Object.entries(props || {}).forEach(([key, value]) => {
+    element[key.toLowerCase()] = value;
+  });
+
+  children.flat().forEach((child) => {
+    if (child instanceof Node) {
+      element.appendChild(child);
+      return;
+    }
+
+    element.appendChild(document.createTextNode(child));
+  });
+
+  return element;
+}
+
+function render(number = 0) {
+  const count = { presentNumber: number };
+
+  const { presentNumber } = count;
+
+  const element = (
+    <div id="hello" className="greeting">
+      <p>Hello, world!</p>
+      <p>
+        <button type="button" onClick={() => render(presentNumber + 1)}>
+          Click me!
+          (
+          {presentNumber}
+          )
+        </button>
+      </p>
+      <p>
+        {[1, 2, 3].map((i) => (
+          <button type="button" onClick={() => render(i)}>
+            {i}
+          </button>
+        ))}
+      </p>
+    </div>
+  );
+
+  document.getElementById('app').textContent = '';
+  document.getElementById('app').appendChild(element);
+}
+
+render(0);
```

Comment:
render 함수에 default value가 있기 때문에 굳이 없어도 될 것 같네요~
```suggestion
render();
```

Reply:
넵 반영하겠습니다! 불필요한 코드는 최대한 제거... noted...

감사합니다 :)